### PR TITLE
User model added

### PR DESCRIPTION
**Pull request by StellarApp**
user model added

### CODE REVIEW

File: backend/data/models/Users.js [hunk 1]
```diff
@@ -2,41 +2,41 @@ const conn = require("./conn");
 const { Sequelize } = conn;
 const { STRING, UUID, UUIDV4 } = Sequelize;
 
-const user = conn.define('user', {
+const user = conn.define("user", {
   id: {
     type: UUID,
     primaryKey: true,
-    defaultValue: UUIDV4,
```
Comment (effyyzhang) [inline backend/data/models/Users.js]: Unique: True

File: backend/data/models/Users.js [hunk 2]
```diff
@@ -2,41 +2,41 @@ const conn = require("./conn");
 const { Sequelize } = conn;
 const { STRING, UUID, UUIDV4 } = Sequelize;
 
-const user = conn.define('user', {
```
Comment (effyyzhang) [inline backend/data/models/Users.js]: User

File: backend/data/models/Users.js [hunk 3]
```diff
@@ -2,41 +2,41 @@ const conn = require("./conn");
...
   email: {
     type: STRING,
     allowNull: false,
     unique: true,
     validate: {
-      len: [1,255]
+      len: [1, 255]
     }
   },
   password: {
```
Comment (effyyzhang) [inline backend/data/models/Users.js]: Add fullName

File: backend/data/models/Users.js [hunk 4]
```diff
@@ -2,41 +2,41 @@ const conn = require("./conn");
 const { Sequelize } = conn;
```
Comment (effyyzhang) [inline backend/data/models/Users.js]: const conn = require('../connection')

File: backend/data/models/Users.js [hunk 5]
```diff
@@ -2,41 +2,41 @@ const conn = require("./conn");
...
 const { STRING, UUID, UUIDV4 } = Sequelize;
 
-const user = conn.define('user', {
+const user = conn.define("user", {
```
Comment (effyyzhang) [inline backend/data/models/Users.js]: const User

File: backend/data/models/Users.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    allowNull: false,
+    validate: {
+      len: [1, 255]
+    }
+  },
+  email: {
+    type: STRING,
+    allowNull: false,
+    unique: true,
+    validate: {
```
Comment (jtanadi) [inline backend/data/models/Users.js]: If not too annoying, under validate, you can add:
`isEmail: true`

**Pull request merged**